### PR TITLE
adds suppression for CVE-2021-37533

### PR DESCRIPTION
**Pull request by amoslv1**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

File: dependency-check-suppressions.xml [hunk 1]
```diff
@@ -22,4 +22,11 @@
     <packageUrl regex="true">^pkg:maven/com\.fasterxml\.jackson\.core/jackson\-databind@.*$</packageUrl>
     <cve>CVE-2022-42003</cve>
   </suppress>
+  <suppress until = "2023-01-01">
+    <notes><![CDATA[
+   file name: commons-fileupload-1.4.jar - this is a mistaken tag for Apache Commons Net
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven/commons\-fileupload/commons\-fileupload@.*$</packageUrl>
```
Comment (TeaWithLucas) [inline dependency-check-suppressions.xml:29]: Needs to be more generic; you can do
```suggestion
    <packageUrl regex="true">^pkg:maven\/commons\-(?!net)\w+\/commons\-(?!net)\w+@.*$</packageUrl>
```

File: dependency-check-suppressions.xml [hunk 2]
```diff
@@ -22,4 +22,11 @@
     <packageUrl regex="true">^pkg:maven/com\.fasterxml\.jackson\.core/jackson\-databind@.*$</packageUrl>
     <cve>CVE-2022-42003</cve>
   </suppress>
+  <suppress until = "2023-01-01">
+    <notes><![CDATA[
+   file name: commons-fileupload-1.4.jar - this is a mistaken tag for Apache Commons Net
+   ]]></notes>
+    <packageUrl regex="true">^pkg:maven\/commons\-(?!net)\w+\/commons\-(?!net)\w+@.*$</packageUrl>
+    <cve>CVE-2021-37533</cve>
+  </suppress>
```
Comment (TeaWithLucas) [inline dependency-check-suppressions.xml:31]:
```suggestion
  <suppress until = "2023-01-01">
    <notes><![CDATA[
     these common libraries are being mistaken as Apache Commons Net
   ]]></notes>
    <packageUrl regex="true">^pkg:maven\/commons\-(?!net)\w+\/commons\-(?!net)\w+@.*$</packageUrl>
    <cve>CVE-2021-37533</cve>
  </suppress>
    <suppress until = "2023-01-01">
    <notes><![CDATA[
     these common libraries are being mistaken as Apache Commons Net
   ]]></notes>
    <packageUrl regex="true">^pkg:maven\/org\-apache\-commons\/commons\-(?!net)\w+@.*$</packageUrl>
    <cve>CVE-2021-37533</cve>
  </suppress>
```

**Review by TeaWithLucas — APPROVED** [review]

File: dependency-check-suppressions.xml (hunk 2 above)
Comment (amoslv1) [reply dependency-check-suppressions.xml]: done

File: dependency-check-suppressions.xml [hunk 3]
```diff
@@ -22,4 +22,11 @@
     <packageUrl regex="true">^pkg:maven/com\.fasterxml\.jackson\.core/jackson\-databind@.*$</packageUrl>
     <cve>CVE-2022-42003</cve>
   </suppress>
+  <suppress until = "2023-01-01">
+    <notes><![CDATA[
+   file name: commons-fileupload-1.4.jar - this is a mistaken tag for Apache Commons Net
```
[the remark this answers is not in the document]
Comment (amoslv1) [reply dependency-check-suppressions.xml]: done

**Review by DGordonV1 — APPROVED** [review]

**Review by nickhill111 — APPROVED** [review]

**Pull request merged**